tests/integration/test_aws/conftest.py

```diff
@@ -204,72 +266,113 @@ def create_test_bucket(buckets_manager,
 
 
 @pytest.fixture
-def manage_bucket_files(metadata: dict):
+def manage_bucket_files(metadata: dict, s3_client, ec2_client):
     """Upload a file to S3 bucket and delete after the test ends.
 
-    Parameters
-    ----------
-    metadata : dict
-        Metadata to get the parameters.
+    Args:
+        metadata (dict): Metadata to get the parameters.
+        s3_client (boto3.resources.base.ServiceResource): S3 client used to manage the bucket resources.
+        ec2_client (Service client instance): EC2 client to manage VPC resources.
```

Comment:
Aren't these the same type?

Reply:
The instanced objects are from different classes. The first one is a [ServiceResource](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.resource) (called `client` since it has a similar use as a client) and the second is a [Service client instance](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.client)